beacon-chain/powchain/deposit.go

```diff
@@ -14,7 +17,7 @@ import (
 
 // processDeposit is a copy of the core function of the same name which includes some optimizations
 // and removes the requirement to pass in beacon state. This is for determining genesis validators.
-func (s *Service) processDeposit(
+func (s *Service) processDepositOld(
```

Comment:
I don't think Old is good here. How about Slow?

Reply:
Hey in my following PR this whole function is removed as it is outdated